quilt/tools/store.py

```diff
@@ -105,6 +109,17 @@ def save_df(self, df, name, path, ext, target):
         """
         raise NotImplementedError()
 
+    def save_file(self, srcfile, name, path, target):
+        """
+        Save a (raw) file to the store.
+        """
+        self._find_path_write()
+        filehash = digest_file(srcfile)
+        fullname = name.lstrip('/').replace('/', '.')
+        self._add_to_contents(fullname, filehash, '', path, target)
+        objpath = os.path.join(self._pkg_dir, self.OBJ_DIR, filehash)
+        copyfile(srcfile, objpath)
```

Comment:
Should we check if `objpath` already exists, just as a tiny optimization (or not so tiny for GB-size files)?